### PR TITLE
Removal of Master Craftsman persona

### PR DESCRIPTION
**Pull request by mholtman**
Here's the pull request for review. 

@mcibulskis @dputman and anyone else: I could use suggestions for how to make the FAQ answer better if you have any.

### CODE REVIEW

File: artisan_roles/faq.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ How does the process of moving up a persona become initiated? [Issue #8](https:/
 : See [Being a Candidate](./progression_candidate.html) for additional information.
 
 Who will judge an artisan's behaviors? [Issue #8](https://github.com/PillarTechnology/guild-docs/issues/8)
-: Artisans will be treated to individual attention, much like the current new-hire interview process. An artisan's mentor and peers are responsible for giving frequent and honest feedback to help the artisan grow. Once the artisan has been identified (Sponsored), evaluations will be given by Master Craftsmen and Craftsmen (your Litmus Test). Depending on the level of the artisan being evaluated, further review of the results of the Litmus Test may be conducted by the Guild Leaders.
+: Artisans will be treated to individual attention, much like the current new-hire interview process. An artisan's mentor and peers are responsible for giving frequent and honest feedback to help the artisan grow. Once the artisan has been identified (Sponsored), evaluations will be given by Craftsmen (your Litmus Test). Depending on the level of the artisan being evaluated, further review of the results of the Litmus Test may be conducted by the Guild Leaders.
```
Comment (kevinknowles-wk) [inline artisan_roles/faq.md]: Do we have a good story around mentors?  I'm not aware of anything formal, but I think that is the perception by some individuals.  Is this the place to address that?  Or just address it on a one on one basis?

Comment (mholtman) [inline artisan_roles/faq.md]: There is nothing formal around mentorship, though it is brought up in every career retro I am a part of. I always ask people if they have a mentor and who they are learning from. I believe that as the larger learning plan for Pillar continues to evolve, that mentorship will be addressed.

File: artisan_roles/faq.md [hunk 2]
```diff
@@ -157,7 +156,7 @@ How will we provide support for artisans to progress without working unsustainab
 Will appropriate and gender neutral allowances be made for candidates at every turn? [Issue #40-41](https://github.com/PillarTechnology/guild-docs/issues/40)
 : The goal of this system is to solve the career progression ambiguity at Pillar. There was not a specific focus on diversity. We welcome any and all input from our Diversity Committee.
 
-Will the master craftsmen have training and diversity goals for individuals and teams? [Issue #41](https://github.com/PillarTechnology/guild-docs/issues/41)
+Will the craftsmen have training and diversity goals for individuals and teams? [Issue #41](https://github.com/PillarTechnology/guild-docs/issues/41)
```
Comment (kevinknowles-wk) [inline artisan_roles/faq.md]: I see this was closed.  Was the resolution to make sure people are getting the appropriate training to help address this?

Comment (mholtman) [inline artisan_roles/faq.md]: I do not know the answer to that question. This PR is strictly to strip the language out of the blog. If we want to revisit issues or do more in the FAQ, that's a different question.

**Pull request merged**